src/python/turicreate/test/test_drawing_classifier.py

```diff
@@ -96,8 +97,20 @@ def setUpClass(self, warm_start=None):
 
     def test_create_with_missing_value_bitmap(self):
         sf = self.check_cross_sf.append(_tc.SFrame({self.feature: _tc.SArray([None], dtype=_tc.Image), self.target: ["check"]}))
-        with self.assertRaises(_ToolkitError):
+        try:
+            _tc.drawing_classifier.create(sf, self.target)
+        except _ToolkitError as e:
+            self.assertTrue("dropna" in str(e))
+
+    def test_create_with_missing_value_in_label(self):
+        sf = self.check_cross_sf
+        sf = sf.remove_column(self.target)
+        sf = sf.add_column(_tc.SArray(
+            [None for _ in range(len(sf))], dtype=str), self.target)
```

Comment:
Rather than `[None for _ in range(len(sf))]` you can just do `[None] * len(sf)`.

Reply:
Wow, will try.